src/app/index.ts

```diff
@@ -18,9 +18,10 @@ const allFunctions = {
   Users: { ...UsersMod },
   Wallets: { ...WalletsMod },
   Payments: { ...PaymentsMod },
-}
+} as const
 
-for (const subModule in allFunctions) {
+let subModule: keyof typeof allFunctions
```

Comment:
yes subModule is really a typed as a string in `for (const subModule in allFunctions) {`

the new code restrict the string so that is can only be the list of string of the function name